pipelines/internal/commands/watch/watch.go

```diff
@@ -55,6 +55,8 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 
 func watch(ctx context.Context, service *genomics.Service, name string) (interface{}, error) {
 	var events []*genomics.Event
+	const initialDelay = 5 * time.Second
+	var delay = initialDelay
```

Comment:
delay := initialDelay (we typically use var only when an initial assignment isn't required)